apps/prairielearn/python/zygote.py

```diff
@@ -122,10 +122,14 @@ def find_spec(
 # helpful because the object - which contains something that cannot be converted
 # to JSON - would otherwise never be displayed to the developer, making it hard to
 # debug the problem.
-def try_dumps(obj: Any, sort_keys=False, allow_nan=False):
+def try_dumps(obj: Any):
     try:
-        zu.assert_all_integers_within_limits(obj)
-        return json.dumps(obj, sort_keys=sort_keys, allow_nan=allow_nan)
```

Comment:
Change in behavior: `orjson` doesn't support throwing if `NaN` is encountered; instead it just serializes it to `null`. This is consistent with how `JSON.stringify(NaN)` behaves in JavaScript, so I think this is a reasonable change. Just calling this out though.